src/zarr/array.py

```diff
@@ -343,6 +343,15 @@ def ndim(self) -> int:
     def shape(self) -> ChunkCoords:
         return self.metadata.shape
 
+    @property
+    def chunks(self) -> ChunkCoords:
+        try:
+            return self.metadata.chunk_grid.chunk_shape  # type: ignore[attr-defined]
```

Comment:
I am fighting with mypy here. My local pre-commit env does not like
```
# type: ignore[attr-defined, no-any-return]
```

but that seems to be what the CI-based pre-commit linter wants

Reply:
I took a swing at fixing this @rabernat. Let me know what you think.